backend/apps/accounts/templates/accounts/email/reset_password.html

```diff
@@ -0,0 +1,13 @@
+{% load i18n %}
+{% load account_extras %}
+
+{% autoescape off %}
+    <h1>{% translate "Reset password" %}</h1>
+
+    <p>{% translate "You're receiving this email because you requested a password reset for your account." %}</p>
```

Comment:
We are sending you this email because you requested a password change.
or: You have received this email because you requested a password change